src/test_merge_sort.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding utf-8 -*-
+import pytest
+import random
+
+def test_include():
+    """test to see if merge_sort can be included"""
+    from merge_sort import merge_sort
+
+
+def test_return_list():
+    """Test to see if a list is returned from the merge_sort function."""
+    from merge_sort import merge_sort
+    my_list = merge_sort([])
+    assert type(my_list) is list
```

Comment:
The preferred way to check that an object's type meets the expected:
`isinstance(object, class)`
e.g.
`isinstance(my_list, list)`